eZ/Bundle/EzPublishRestBundle/Resources/config/routing.yml

```diff
@@ -95,6 +95,15 @@ ezpublish_rest_copyContent:
     requirements:
         contentId: \d+
 
+ezpublish_rest_deleteContentTranslation:
+    path: /content/objects/{contentId}/{languageCode}
+    defaults:
+        _controller: ezpublish_rest.controller.content:deleteContentTranslation
+    methods: [DELETE]
+    requirements:
+        contentId: \d+
+        languageCode: '[a-z]{3}-[A-Z]{2}'
```

Comment:
Ping @andrerom. Don't you think it would be better to update specs and use `/content/objects/{contentId}/translations/{languageCode}` as @alongosz suggested?